wom/src/main/scala/wdl/WdlNamespace.scala

```diff
@@ -563,6 +571,23 @@ object WdlNamespace {
       throw new IllegalArgumentException(s"$str is not a valid import")
     }
   }
+
+  def httpResolver(str: String): WorkflowSource =
+    httpResolverWithHeaders(Map.empty[String,String])(str)
+
+  def httpResolverWithHeaders(headers: Map[String,String])(str: String): WorkflowSource = {
+    implicit val sttpBackend = AsyncHttpClientCatsBackend[IO]()
+
+    val responseIO : IO[Response[String]] =
+      sttp.get(uri"$str").headers(headers)send()
```

Comment:
I don't get this syntax...

Reply:
huh... me neither, looks like the "." went missing but it somehow still works.  I'll put the . back